docs/tutorials/time-series.md

```diff
@@ -0,0 +1,766 @@
+# Time Series Analysis with Feldera
+
+This guide covers concepts required to effectively process **time series**
+data with Feldera.  A time series is a sequence of events, such as IoT sensor
+readings or financial transactions, where each event is associated with one or
+more timestamps.  Time series data is inherently dynamic, changing continuously
+as new events arrive in real-time.  Feldera is designed to
+efficiently compute over changing data, making it well-suited for time series
+analytics.
```

Comment:
Explain what we mean by efficiently. I don't think most people think in terms of unbounded streams + finite state/memory, which is the killer feature here.